MTM-44129 Added related topics to Cockpit section (#594)
SoftwareAG/c8y-docs: content/users-guide/cockpit-bundle/widgets-collection.md

layout: redirect
---

{{< c8y-admon-related >}}

> - [Device Management](/users-guide/device-management#overview) in the *User guide* for details on working with devices.

Reviewer: I'm not sure if it makes sense to point to the DM users guide... A lot of the sections are somehow connected to devices, why show this here exactly?

Reviewer 2: I agree.

> - [Device Management](/users-guide/device-management#overview) in the *User guide* for details on working with devices.

Reviewer: Just to add to this, maybe it makes sense to point to the DM user guide in the data-explorer section as it shows data incoming from devices.

Author: @eniosultan Do you mean the Data explorer section in the Cockpit section?

Reviewer: @MWindel  yes, exactly 👍

Author: ```suggestion
- [Cockpit > Data explorer](/users-guide/cockpit#data-explorer) in the *User guide* for details on visualizing your data when working with widgets.
```

- [Administration](/users-guide/administration#managing-applications) in the *User guide* for details on managing applications in {{< product-c8y-iot >}}.

> - The [Protocol integration guide](/protocol-integration/cloud-fieldbus/#overview) for more information on using Cloud Fieldbus in {{< product-c8y-iot >}}.

Reviewer 3: many occurrences of "... in Cumulocity IoT." towards the end of the related topics list items - maybe we can reduce them a little bit?

layout: redirect
---

{{< c8y-admon-related >}}
- [Device Management](/users-guide/device-management#overview) in the *User guide* for details on working with devices.
- [Administration](/users-guide/administration#managing-applications) in the *User guide* for details on managing applications.

> - The [Protocol integration guide](/protocol-integration/cloud-fieldbus/#overview) for more information on using Cloud Fieldbus in {{< product-c8y-iot >}}.

Reviewer 2: ```suggestion
- [Cloud Fieldbus](/protocol-integration/cloud-fieldbus/#overview) in the *Protocol integration guide* for more information on using Cloud Fieldbus in {{< product-c8y-iot >}}.
```

> - The [Web SDK guide > Tutorials > Add a custom widget to a dashboard](/users-guide/web/tutorials/#add-a-custom-widget) for further details for customized widgets in the {{< product-c8y-iot >}} environment.

Reviewer 2: ```suggestion
- [Tutorials > Add a custom widget to a dashboard](/users-guide/web/tutorials/#add-a-custom-widget) in the *Web SDK guide* for further details on customized widgets in the {{< product-c8y-iot >}} environment.
```

> - [Tutorials > Add a custom widget to a dashboard](/users-guide/web/tutorials/#add-a-custom-widget) in the *Web SDK guide* for further details on customized widgets in the {{< product-c8y-iot >}} environment.

Reviewer 2: ```suggestion
- [Tutorials > Add a custom widget to a dashboard](/users-guide/web/tutorials/#add-a-custom-widget) in the *Web SDK guide* for details on customized widgets in the {{< product-c8y-iot >}} environment.
```
